ci/templates/jobs-base.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- job:
+    name: cifmw-base-crc-pre-post
+    nodeset: centos-9-crc-xxl
+    abstract: true
+    vars:
+      crc_parameters: "--memory 16000 --disk-size 120 --cpus 6"
+    parent: base-crc
+    pre-run:
+      - ci/playbooks/e2e-prepare.yml
+      - ci/playbooks/dump_zuul_vars.yml
+    post-run: ci/playbooks/collect-logs.yml
+
+- job:
+    name: cifmw-base-crc-kuttl
+    abstract: true
+    parent: cifmw-base-crc-pre-post
+    run:
+      - ci/playbooks/kuttl-run.yml
```

Comment:
This playbook is missing. I assume the content of this playbook will look like this.
```
- hosts: controller
  gather_facts: true
  tasks:
    - name: Run kuttl tests playbook
      ansible.builtin.command:
        chdir: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}"
        cmd: >-
          ansible-playbook ci/playbooks/e2e-kuttl.yml
          -e @scenarios/centos-9/base.yml
          -e @scenarios/centos-9/ci.yml
          -e @scenarios/centos-9/zuul_inventory.yml

```